core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -35,11 +36,14 @@ import (
 )
 
 const (
+	defaultPluginRetryExpiration = 5 * time.Minute
```

Comment:
probably set this higher? I'm thinking we should keep this until a pipeline run is eligible, i.e. check block isn't too old. 128 blocks on eth would be ~25 mins, so maybe let's keep this to 30m?

Reply:
done